### PR TITLE
added description for manual sorting

### PR DESCRIPTION
**Pull request by bluedeepimpact**
[ci skip]

### CODE REVIEW

**Review by RobinWagner — APPROVED** [review]
Looks good, thanks.

**Review by RobinWagner — APPROVED** [review]
Looks good 👍

File: docs/user-guide/work-packages/work-package-table-configuration/README.md [hunk 1]
```diff
@@ -99,6 +100,13 @@ The same filter applied in the hierarchy mode.
 
 ![sort-hierarchy-mode](sort-hierarchy-mode.png)
 
+### Manual sorting of the work package list
+
+You can sort the work package list manually, using the icon with 6 dots on the left of each work package to drag and drop it.
```
Comment (RobinWagner) [inline docs/user-guide/work-packages/work-package-table-configuration/README.md]: I would add a "the" before "6 dots".

**Review by RobinWagner — APPROVED** [review]
Looks good. Thanks.
I just added one comment for a minor change.

**Comment by bluedeepimpact** [comment]
merging after review by RobinWagner

**Pull request merged**